tests/unit/components/sl-button-test.js

```diff
@@ -10,130 +14,76 @@ const mockStreamService = {
     send() {}
 };
 
-test( 'Label changes for pending state', function( assert ) {
-    const pendingText = 'Pending';
-    const staticText = 'Static';
-    const component = this.subject({
-        pendingLabel: pendingText,
-        label: staticText
-    });
-
-    assert.equal(
-        component.get( 'currentLabel' ),
-        staticText,
-        'Static text is set initially'
-    );
+test( 'Successfully mixed sl-stream-enabled', function( assert ) {
+    const testObject = Ember.Object.extend( streamEnabled );
+    const subject = testObject.create();
 
-    Ember.run( () => {
-        component.set( 'pending', true );
-    });
-
-    assert.equal(
-        component.get( 'currentLabel' ),
-        pendingText,
-        'Pending text is set while pending'
+    assert.ok(
+        subject
```

Comment:
Will need to be in sync with the decision reached in https://github.com/softlayer/sl-ember-components/pull/798#issuecomment-139342495